src/plugins/auto_batch/tests/functional/behavior/executable_network/exec_net_base.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/executable_network/exec_network_base.hpp"
+
+using namespace BehaviorTestsDefinitions;
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP32,
+                                                               InferenceEngine::Precision::FP16};
+auto autoBatchConfig = []() {
+    return std::vector<std::map<std::string, std::string>>{
+        // explicit batch size 4 to avoid fallback to no auto-batching (i.e. plain GPU)
+        {{CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG), std::string(CommonTestUtils::DEVICE_GPU) + "(4)"},
```

Comment:
I suppose AUTO_BATCH should be tested without help of any other HW plugin, let's just use mock plugins (as in caching tests).
I understand that in some cases we still need to perform inference to check runtime part of AUTO_BATCH. For this purpose we can use TEMPLATE plugin.